components/public-api/gitpod/experimental/v1/pagination.proto

```diff
@@ -5,9 +5,9 @@ package gitpod.experimental.v1;
 option go_package = "github.com/gitpod-io/gitpod/components/public-api/go/experimental/v1";
 
 message Pagination {
-    // page_size is the maximum number of results we expect
+    // page_size is the maximum number of results to retrieve per page
     int32 page_size = 1;
 
-    // page_token points to a specific page of the results
-    string page_token = 2;
+    // page is the page number of results to retrieve.
+    string page = 2;
```

Comment:
Why is this change necessary? Does it affect other services?

Reply:
Unused by other services at this point. Originally we designed as per Google API design recommendations but our existing APIs don't deal with "next_tokens" so changed it to what we do deal with today.